contents/docs/cdp/destinations/airtable.md

```diff
@@ -36,7 +36,7 @@ You should have these details now:
 With them, we’re ready to set up the Airtable destination.
 
 1. In PostHog, click the [Data pipeline](https://us.posthog.com/pipeline/overview) tab in the left sidebar.
-2. Click the **Destinations** tab.
+2. Click the [Destinations](https://us.posthog.com/pipeline/destinations?search=) tab.
```

Comment:
Since we can use search params, any reason not to (for all of them)? 
```suggestion
2. Click the [Destinations](https://us.posthog.com/pipeline/destinations?search=airtable) tab.
```